libs/api/domains/national-registry-x-road/src/dto/nationalRegistryChildGuardianshipInput.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
+export class GetChildGuardianshipInput {
```

Comment:
This input's name is not referencing the domain it corresponds to. Most likely best to correspond the name to the resolver that uses this input, thus the name could be:
```typescript
NationalRegistryUserV2ChildGuardianshipInput
```